exceptions_sample.py
log.txt